### ISSUE TITLE
NameError : global name undefined

### ISSUE THREAD

**Issue by ChaiBapchya** [edgi-govdata-archiving/web-monitoring-processing#31]
File `PageFreezer.py` - refers to global name a
It hasn't been defined anywhere
```
Traceback (most recent call last):
  File "pagefree.py", line 12, in <module>
    print pf.full_html_changes()
  File "/home/chai/GSOC/PageFreezer.py", line 44, in full_html_changes
    display(HTML(a['output']['html']))
NameError: global name 'a' is not defined
```

**Comment by ChaiBapchya** [comment]
1. Either we could - create global variable a as
`a = self.query_result`

Or
2. We could replace `a` with `self.query_result`

**Comment by janakrajchadha** [comment]
@ChaiBapchya Thanks for the suggested solutions.
I had also faced this issue and the other two issues (api key, encoding) when I had tried to run the pagefreezer python module script on my machine. While I was able to solve these issues, I believe there's some level of ambiguity in some other lines of code as well and maybe we can ask the person who had made the original commit to add a few more comments to the code. I had previously raised an issue as well- #24
